dockstore-integration-testing/src/test/java/io/dockstore/webservice/ServiceIT.java

```diff
@@ -178,7 +179,7 @@ public void testProprietaryAPI() {
      * @param path         Path of endpoint
      */
     private void testXTotalCount(Client jerseyClient, String path) {
-        Response response = jerseyClient.target(path).request().get();
+        Response response = jerseyClient.target(path).request().property(ClientProperties.READ_TIMEOUT, 0).get();
```

Comment:
Just curious, why did this need to change?

Reply:
Without it, one of the tests can timeout.
It is possible the slower DB was responsible for slowing things to the extent it timed out, but I figure it couldn't hurt to keep it.